### PR TITLE
lesson_11

### PR DESCRIPTION
**Pull request by NataKanyuk**

### CODE REVIEW

File: lesson_11/index.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    <title>Flower shop</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <div class="work_time">
+        <p>Mon-Fri:07:00-21:00</p>
+        <p>Sat-San:09:00-18:00</p>
+      </div>
+      <img src="logo.png" />
```
Comment (RomanIlkiv) [inline lesson_11/index.html]: don't forget about `alt` attribute

File: lesson_11/index.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+          <div>
+            <img src="rose.png" />
+            <span>ROSE</span>
+          </div>
+          <div class="flower">
+            <img src="lilies.png" />
+            <span>LILIES</span>
+          </div>
+          <div class="flower">
+            <img src="Orchid .png" />
```
Comment (RomanIlkiv) [inline lesson_11/index.html]: redundant space before `.png`

File: lesson_11/index.html [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+            <span>ORCHID </span>
```
Comment (RomanIlkiv) [inline lesson_11/index.html:44]: here the same

File: lesson_11/index.html [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+          <div class="flower">
+            <img src="hydrangea.png" />
+            <span>HYDRANDEA </span>
+          </div>
+          <div class="flower">
+            <img src="tulips.png" />
+            <span>TULIPS</span>
+          </div>
+          <div class="flower">
+            <img src="Peony.png" />
```
Comment (RomanIlkiv) [inline lesson_11/index.html]: no `alt` attributes anywhere

File: lesson_11/dried flowers.png
Comment (RomanIlkiv) [inline lesson_11/dried flowers.png:1]: move all images to separate folder
and fonts too

**Review by RomanIlkiv — CHANGES_REQUESTED** [review]

**Review by NataKanyuk — COMMENTED** [review]
I made changes and fixed the error

**Pull request merged**